src/workspace-data/data-table/wds/AttributeInput.ts

```diff
@@ -0,0 +1,122 @@
+import _ from 'lodash/fp';
+import { Fragment, useEffect, useRef } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { Link } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import * as Utils from 'src/libs/utils';
+
+import { renderInputForAttributeType } from '../shared/AttributeInput';
+import { getAttributeType } from './attribute-utils';
+
+const defaultValueForAttributeType = (attributeType: string | undefined) => {
```

Comment:
It would be nice to have to a return type annotated on this function.

```suggestion
const defaultValueForAttributeType = (attributeType: string | undefined): any => {
```